src/features/embalm/stepContent/utils/errors.ts

```diff
@@ -0,0 +1,46 @@
+import { Archaeologist } from '../../../../types';
+import { CreateSarcophagusStage } from './createSarcophagus';
+import { formatContractCallException } from '../../../../lib/utils/contract-error-handler';
+
+const processArchCommsException = (offendingArchs: Archaeologist[]) => {
+  if (!!offendingArchs.length) {
+    console.log(
+      '',
+      offendingArchs.map(
+        a => `${a.profile.peerId}:\n ${a.exception!.code}: ${a.exception!.message}`
+      )
+    );
+  }
+};
+
+export const createSarcophagusErrors: Record<number, string> = {
+  [CreateSarcophagusStage.DIAL_ARCHAEOLOGISTS]: 'Could not connect to all archaeologists',
+  [CreateSarcophagusStage.UPLOAD_ENCRYPTED_SHARDS]: 'Upload Archaeologist Data to Arweave',
+  [CreateSarcophagusStage.ARCHAEOLOGIST_NEGOTIATION]: 'Retrieve Archaeologist Signatures',
+  [CreateSarcophagusStage.UPLOAD_PAYLOAD]: 'Upload File Data to Arweave',
```

Comment:
Likely copy-paste oversight -- all string values after `DIAL_ARCHAEOLOGISTS` aren't error messages

Reply:
Good catch -- looks like the first one is the only one being used currently (but this should get updated) -- I updated them all to be actual error messages.